benchmark/fs/readfile-partitioned.js

```diff
@@ -47,10 +48,14 @@ function main({ len, dur, concurrent }) {
     } catch {
       // Continue regardless of error.
     }
-  }, dur * 1000);
+  }, duration * 1000);
 
   function read() {
-    fs.readFile(filename, afterRead);
+    if (encoding !== '') {
```

Comment:
I don't think we need this. An empty string for the encoding results in Buffers passed to the callback, the same as excluding the argument entirely.

Reply:
Thanks for the feedback. I updated my pull request.